### PR TITLE
Save next step of enrollment flow

### PR DESCRIPTION
**Pull request by lionick**
ALTER TABLE cm_co_petitions ADD COLUMN coef_next_step VARCHAR(256);

### CODE REVIEW

File: app/Model/CoPetition.php [hunk 1]
```diff
@@ -158,7 +158,11 @@ class CoPetition extends AppModel {
                                       PetitionStatusEnum::PendingApproval,
                                       PetitionStatusEnum::PendingConfirmation)),
       'required' => true,
-      'message' => 'A valid status must be selected'
+      'message' => 'A valid status must be selected',
+      'coef_next_step' => array(
```
Comment (ioigoume) [inline app/Model/CoPetition.php]: `coef_next_step` need to be a distinct validation rule and not part of the `status` field rule

**Review by ioigoume — CHANGES_REQUESTED** [review]

**Review by ioigoume — APPROVED** [review]
LGTM

**Review by ioigoume — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**